packages/plugin-core/package.json

```diff
@@ -754,12 +772,6 @@
     "markdown.previewStyles": [
       "./media/fontello/css/fontello.css",
       "./media/markdown.css"
-    ],
-    "yamlValidation": [
```

Comment:
this is also a miss from our part. can you hardcode this in the [genConfig](https://github.com/dendronhq/dendron/blob/6c98d466536632530399bd45f1220ae725ff3e2f/packages/plugin-core/scripts/genConfig.ts#L86-L86) script?